contracts/governance/TimelockController.sol

```diff
@@ -223,6 +228,9 @@ contract TimelockController is AccessControl, IERC721Receiver, IERC1155Receiver
         bytes32 id = hashOperation(target, value, data, predecessor, salt);
         _schedule(id, delay);
         emit CallScheduled(id, 0, target, value, data, predecessor, delay);
+        if (salt != bytes32(0)) {
+            emit ProposalSalt(id, salt);
```

Comment:
The functoin documentation says that `CallScheduled` is emitted. We need to add `ProposalSalt` in there as well. Same in `scheduleBatch`.